workbench.js

```diff
@@ -394,12 +447,16 @@ Workbench.prototype.waitForReceipt = function (txHash) {
   });
 };
 
+function checkForSandboxReceiptErrors(receipt, txHash) {
+  if (receipt.exception === 'out of gas') console.log('Out of gas: tx ' + txHash);
+}
 Workbench.prototype.waitForSandboxReceipt = function (txHash) {
   var self = this;
   return new Promise((resolve, reject) => {
     var called = false;
     function cb(err, receipt) {
       if (err) return reject(err);
+      checkForSandboxReceiptErrors(sandboxReceipt, txHash);
```

Comment:
I think I didn't commit everything, let me check

On Mon, Aug 29, 2016, 14:03 Alexander Sinyagin <notifications@github.com>
wrote:

> In workbench.js
> <https://github.com/ether-camp/ethereum-sandbox-workbench/pull/7#discussion_r76589473>
> :
>
> >  Workbench.prototype.waitForSandboxReceipt = function (txHash) {
> >    var self = this;
> >    return new Promise((resolve, reject) => {
> >      var called = false;
> >      function cb(err, receipt) {
> >        if (err) return reject(err);
> > +      checkForSandboxReceiptErrors(sandboxReceipt, txHash);
>
> I get an error here because sandboxReceipt is not defined. I think it
> should be receipt.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ether-camp/ethereum-sandbox-workbench/pull/7/files/13a4c59524d983104141834823701b1a1bfef3df#r76589473>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADW2GM4WuKNl2Ua03c-BzEMkclpliNesks5qkryXgaJpZM4Ju41B>
> .
>